app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::MessagesController < ApplicationController
+  def index
+    @message = Message.all.sample(1)
```

Comment:
Please ensure that your `API` endpoint returns a random greeting according to the project requirement.